src/Bicep.LangServer/Handlers/BicepDeploymentParametersHandler.cs

```diff
@@ -148,7 +148,10 @@ private bool IsExpression(SyntaxBase modifier)
             return modifier is ParameterDefaultValueSyntax parameterDefaultValueSyntax &&
                 parameterDefaultValueSyntax.DefaultValue is ExpressionSyntax expressionSyntax &&
                 expressionSyntax is not null &&
-                expressionSyntax is not StringSyntax &&
+                (
+                    expressionSyntax is not StringSyntax ||
+                    expressionSyntax is StringSyntax specificationString && specificationString.IsInterpolated()
```

Comment:
nit: Could you please add a comment or link to issue? Thanks!

Reply:
I think this simpler syntax would work:

(expressionSyntax is not StringSyntax || specificationString.IsInterpolated())

(and remove the expressionSyntax is not StringSyntax line)